src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SentenceCalculationService.kt

```diff
@@ -204,7 +156,7 @@ class SentenceCalculationService(private val sentenceAdjustedCalculationService:
   ): ReleaseDateCalculation {
```

Comment:
I know this is not new with this PR, but the getSingleSentenceRelease function has an unused param (booking), can we remove it?

Reply:
Done